### ISSUE TITLE
Kernel Dense_T8x8_R8x8 is missing when evaluating model

### ISSUE THREAD

**Issue by fmannhardt** [Unity-Technologies/barracuda-release#2]
I am trying out the new release 0.3 of Barracuda and got an error `Kernel Dense_T8x8_R8x8 is missing` when trying to run a standard CNN model (Relu, Conv2D, MaxPool, Softmax) that was converted to ONNX through the provided Docker scripts.

I looked at the source and this issue could be solved by uncommenting line 6 here:
https://github.com/Unity-Technologies/barracuda-release/blob/4a4217524e559fcf65440bded8f2c8a70c8c8c07/Barracuda/Resources/Dense.compute#L6

I now found another workaround, which is to convert from a Tensorflow model (pb file) instead of the ONNX model, but maybe this is not supposed to be commented out?

**Comment by mantasp** [comment]
@fmannhardt any chance you could share your model for testing this issue?

**Comment by fmannhardt** [comment]
I tried to reproduce the issue with the same model re-initalized with random weights. However, I have trouble reproducing it since this was an older model and it seems the conversion pipeline from Tensorflow to ONNX has changed and this issue is gone (some others turned up, but I will open separate issues when I got them narrowed down).

Sorry about not being able to provide some examples for this issue. Maybe this was just a weird combination of outdated converters.
